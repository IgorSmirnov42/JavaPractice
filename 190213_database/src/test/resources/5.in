1
a
239
1
b
42
5
a
239
b
5
b
42
t
7
0
