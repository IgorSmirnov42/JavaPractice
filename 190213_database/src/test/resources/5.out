Write command name (8 for help): Write owner name: Write phone number: Write command name (8 for help): Write owner name: Write phone number: Write command name (8 for help): Write owner name: Write new owner name: Write command name (8 for help): Unknown command name
Write command name (8 for help): Write owner name: Write new owner name: Write command name (8 for help): Unknown command name
Write command name (8 for help): 239 239
42 42
Write command name (8 for help): 